build.gradle

```diff
@@ -27,9 +27,14 @@ subprojects {
         maven {
             url "https://oss.sonatype.org/content/repositories/snapshots"
         }
+        maven {
+            url "https://repo.spongepowered.org/maven"
+        }
     }
 
     dependencies {
+        compileOnly "org.spigotmc:spigot-api:1.14-R0.1-SNAPSHOT"
+        compileOnly "org.spongepowered:spongeapi:7.1.0"
```

Comment:
Remove these from the parent - reason being is so that the autocomplete is nicer, we don't accidentally use sponge in bukkit and bukkit in sponge, etc etc etc etc.